awx/ui/client/src/organizations/linkout/addUsers/addUsers.controller.js

```diff
@@ -23,18 +23,32 @@ function($scope, $rootScope, ProcessErrors, GetBasePath, generateList,
             $scope.addType = "Administrators";
         }
 
-        init();
+        let notAdminAlreadyParams = {};
+
+        if ($scope.addType === 'Administrators') {
+            Rest.setUrl(GetBasePath('organizations') + `${$state.params.organization_id}/object_roles`);
+            Rest.get().then(({data}) => {
+                notAdminAlreadyParams.not__roles__in = data.results
+                    .filter(({name}) => name === i18n._('Admin'))
+                    .map(({id}) => id)
+                    .join(',');
+                notAdminAlreadyParams.is_superuser = 'false';              
+                init();
+            });
+        } else {
+            init();
+        }
 
         function init(){
-            $scope.add_user_default_params = {
+            $scope.add_user_default_params = Object.assign({
                 order_by: 'username',
                 page_size: 5
```

Comment:
I think explicitly declaring `is_superuser = 'false'` within the `add_user_default_params` object is more legible than using `Object.assign()` to add it implicitly, unless there is some reason I am I missing.

Reply:
👍 I had this sort of "merge two objects" pattern based on where some code lived before, but where this ended up it would definitely be a clean up if I moved it.